internal/ingress/annotations/authtls/main_test.go

```
@@ -165,8 +170,11 @@ func TestAnnotations(t *testing.T) {
 	if u.PassCertToUpstream != true {
 		t.Errorf("expected %v but got %v", true, u.PassCertToUpstream)
 	}
-	if u.MatchCN != "CN=hello-app" {
-		t.Errorf("expected %v but got %v", "CN=hello-app", u.MatchCN)
+	if u.ExactMatchCN != "hello-app" {
+		t.Errorf("expected %v but got %v", "hello-app", u.ExactMatchCN)
+	}
+	if match, _ := regexp.MatchString(u.RegexMatchCN, "hello"); !match {
```

please don't ignore the error here